config/deploy.rb

```diff
@@ -10,7 +10,7 @@
 set :deploy_to, '/opt/app/dor_indexer/dor_indexing_app'
 
 set :linked_dirs, %w[log config/settings tmp/pids tmp/cache tmp/sockets vendor/bundle]
-set :linked_files, %w[config/secrets.yml config/honeybadger.yml config/newrelic.yml]
+set :linked_files, %w[config/honeybadger.yml]
```

Comment:
is config/secrets.yml in shared_configs?  should it be removed if so?

Reply:
It was removed as part of the secrets-to-vault mini-patch party.